site/lib/moralis.ts

```diff
@@ -0,0 +1,9 @@
+import Moralis from "moralis/node.js";
+
+export const MoralisClient = Moralis.start({
+  appId: process.env.NEXT_PUBLIC_MORALIS_APP_ID,
```

Comment:
I've made the following secrets available under `process.env`
![image](https://user-images.githubusercontent.com/88635679/161162537-3ac24f4a-58e0-4ebd-b1c4-13e4ac8185ab.png)

Reply:
@Atmosfearful 
We'll need to instantiate the client with `MORALIS_APP_ID` and `MORALIS_SERVER_URL`.
Do we need the `NEXT_PUBLIC` prefix too expose it to the client?

Reply:
Ah derp, will do that now.

Reply:
![image](https://user-images.githubusercontent.com/88635679/161637782-cf504e0b-feef-41ff-bbd8-53027375c2ac.png)